namer/namer.cc

```diff
@@ -200,7 +200,7 @@ class FoundDefinitions final {
     }
 
     FoundDefinitionRef addSymbol(core::SymbolRef symbol) {
-        return FoundDefinitionRef(DefinitionKind::Symbol, symbol._id);
+        return FoundDefinitionRef(DefinitionKind::Symbol, symbol.rawId());
```

Comment:
Is it important that a FoundDefinitionRef use the `u4` ID? could it store a `SymbolRef` directly instead of having to unwrap it?

Reply:
iirc, the field storing ID has a different purpose depending on the Kind.